packages/clerk-js/src/ui/elements/UserAvatar.tsx

```diff
@@ -19,8 +19,8 @@ export const UserAvatar = (props: UserAvatarProps) => {
     <Avatar
       title={generatedName}
       initials={initials}
-      imageUrl={profileImageUrl}
       {...rest}
+      imageUrl={__CLERK_FF_ENABLE_IMAGES__ ? imageUrl : profileImageUrl}
```

Comment:
❓ shouldn't we leave it in the same order as it was? Like this:
```typescript
      imageUrl={__CLERK_FF_ENABLE_IMAGES__ ? imageUrl : profileImageUrl}
      {...rest}
```

Reply:
Shouldn't matter i think :) but Im going to revert